salt/modules/pip.py

```diff
@@ -1154,6 +1171,8 @@ def list_(prefix=None,
           user=None,
           cwd=None,
           env_vars=None,
+          user_install=False,
+          verbose=False,
```

Comment:
Where did this come from? Part of the merge conflict resolution maybe? Need to figure out where the docs forthis went or add them if you added this.